src/components/Header/HeaderContact.js

```diff
@@ -90,12 +90,16 @@ const styles = StyleSheet.create({
   }),
   text: {
     color: COLORS.black,
-    fontFamily: fonts.inter[600],
+    fontFamily: fonts.inter[500],
     fontSize: normalizeFontSize(14),
-    fontWeight: 'bold',
     padding: 10,
     textAlign: 'center'
   },
+  titleText: {
+    position: 'absolute',
+    width: '100%',
+    zIndex: -1
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a new style property `titleText` with absolute positioning could potentially overlap with other elements. Ensure that this does not disrupt the layout or visibility of any other components on the screen.
```diff
+  titleText: {
+    position: 'absolute',
+    width: '100%',
+    zIndex: -1
+  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->